src/_data/cacheBust.js

```diff
@@ -0,0 +1,20 @@
+const md5File = require('md5-file');
```

Comment:
Is this included into production bundle?

Reply:
Used only while build for file names